lib/ukrust/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
This commit - "lib/ukrust: Update ukrust to work alongside Cargo" - and the previous one "lib/nolibc: Add bcmp" don't seem related to the PR topic, possibly they are part of a rebase.